docker/Dockerfile.base.16.04

```diff
@@ -51,9 +51,6 @@ RUN \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
-# Build GLFW because the Ubuntu 16.04 version is too old
```

Comment:
hmm please don't remove these breadcrumbs explaining why we bother building glfw from source

Reply:
Done